Documentation/devel/quickstart-dev.md

```diff
@@ -0,0 +1,97 @@
+# Preparing development environment and first rkt build
+This is an example configuration for fresh installation of Ubuntu 16.04 GNOME.
+This doc is also short quick start guide. For more accurate click [here](https://github.com/coreos/rkt/blob/master/Documentation/hacking.md)
+### Required software
+On fresh system install curl and git:
+```sh
+$ sudo apt-get install git curl
+```
+
+### Get rkt repo
+```sh
```

Comment:
Previously, there was a short intro here explaining that `~/Repos` is just a convenience path for personal workspace arrangement. I think you only needed to reword it instead of removing it, this now feels a bit incomplete/unexplained.